src/telemetry/common.ts

```diff
@@ -1,19 +1,16 @@
 import { FastifyRequest } from 'fastify';
-import dc from 'node:diagnostics_channel';
-import { SemanticAttributes as SemAttr } from '@opentelemetry/semantic-conventions';
+export { default as dc } from 'node:diagnostics_channel';
+export * from '@opentelemetry/semantic-conventions';
```

Comment:
Why are we re-exporting 3party packages?

Reply:
If we were to do it I would rather make it an abstraction layer/class/function that hides implementation and allows easy change without refactor. If its just due shorter lines I would rather keep it as is.

Reply:
If we were to do it I would rather make it an abstraction layer/class/function that hides implementation and allows easy change without refactor. Just changing imports I would rather leave it as is.

Reply:
I'll remove the re-export of these two

Reply:
:triggered: it doesn't want to give me a suggestion for auto-import of them
<img width="366" alt="image" src="https://github.com/user-attachments/assets/97f7fdf9-2308-4182-95fe-3a7a51ccdc32">

Reply:
Needed to be a direct dependency for auto suggestions to work 😬